src/main/java/agh/project/GUI/GuiElement.java

```diff
@@ -0,0 +1,267 @@
+package agh.project.GUI;
+
+import agh.project.Map.AbstractWorldMap;
+import agh.project.Elements.Animal;
+import agh.project.Elements.Vector2d;
+import agh.project.Elements.IMapElement;
+import agh.project.Map.IWorldMap;
+import javafx.geometry.HPos;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.*;
+import javafx.scene.paint.Color;
+import javafx.stage.Stage;
+
+
+import java.io.*;
+
+import static java.lang.System.out;
+
+public class GuiElement extends AbstractWorldMap {
+    private Image imageRed = null;
+    private Image imageOrange = null;
+    private Image imageLightOrange = null;
+    private Image imageGold = null;
+    private Image imageBrown = null;
+    private Image imageGreen = null;
+
+    public GuiElement() {
```

Comment:
proponowałbym nie trzymać się sztywno nazw z laborek, bo ta klasa robi zupełnie co innego - to nie jest wizualna reprezentacja jednego zwierzątka tylko całej siatki